src/libos/src/lib.rs

```diff
@@ -54,6 +54,9 @@ extern crate serde_json;
 #[macro_use]
 extern crate memoffset;
 
```

Comment:
Remove the blanket line here so that all crate declarations are together.

Reply:
Done